specification/adp/resource-manager/readme.python.md

```diff
@@ -26,10 +27,10 @@ clear-output-folder: false
 perform-load: false
 ```
 
-``` yaml $(tag) == 'package-2020-07-01-preview'
-namespace: azure.mgmt.adp.v2020_07_01_preview
-output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp/v2020_07_01_preview
+``` yaml
+namespace: azure.mgmt.adp.v$(nstag)
+output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp/v$(nstag)
 python:
-  namespace: azure.mgmt.adp.v2020_07_01_preview
-  output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp/v2020_07_01_preview
+  namespace: azure.mgmt.adp.v$(nstag)
+  output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp/v$(nstag)
```

Comment:
And we could just hard code the `2020_07_01_preview` here instead of using the `nstag` variables